README.md

```diff
@@ -82,7 +84,13 @@ For a future use you can omit the second step and run `up` (the third step) dire
 ### Windows
 
 Currently not supported, but should work using docker as well.
+### Bitcoin regtest
+Optionally you can run bitcoin regtest backend with trezor-user-env. This will run another docker container that contains bitcoind in regtest mode and our blocbook backend connected to that bitcoind in regtest mode.
+#### Run it
 
+1. Clone this repo `git clone git@github.com:trezor/trezor-user-env.git`.
+2. Download the latest docker build: `docker-compose -f ./docker/compose.yml pull trezor-user-env-regtest`
+3. Run it: `docker-compose -f ./docker/compose.yml up trezor-user-env-regtest`
```

Comment:
We could also have the section with commands running both containers at the same time - but that would depend on the usage, if regular users would be using the regtest functionality often

Reply:
Ok let's do it that way then